packages/create-next-app/index.ts

```diff
@@ -358,7 +358,25 @@ async function run(): Promise<void> {
           active: 'Yes',
           inactive: 'No',
         })
+
         program.experimentalApp = Boolean(appDir)
+
+        if (program.experimentalApp) {
+          const nodeVersion = process.version
+          const [major, minor, _patch] = nodeVersion.split('v')[1].split('.')
+
+          if (!(Number(major) >= 16 && Number(minor) >= 8)) {
+            console.log(
+              chalk.red(
+                'The experiemental `app/` directory requires at least Node v16.8.\n' +
+                  'Found ' +
+                  nodeVersion +
+                  ' instead.'
+              )
+            )
+            process.exit(1)
```

Comment:
Hmm I'm not sure this check is explicitly needed as it will be checked on bootup and Node.js v14 is about to become EOL in a couple months anyways and then we can bump the engines value x-ref: https://github.com/nodejs/Release

Reply:
It's to at least guide people why they're encountering issues when trying to run `yarn dev` after spinning up the app because of their node version. Though there's only 2 of use that I know, the other one is [here](https://github.com/vercel/next.js/discussions/47165)